tsconfig.node-16-base.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "./tsconfig.node-base.json",
+  "compilerOptions": {
+    "module": "node16",
+    "moduleResolution": "node16",
+  }
+}
```

Comment:
why do you need a separate tsconfig, based specifically on node16?

Reply:
This is needed for the build command in the root `package.json`. If we use the normal tsconfig, the build fails as the project is of type module and uses ES modules.